haddock-api/resources/html/Ocean.std-theme/ocean.css

```diff
@@ -73,7 +73,7 @@ body.js-enabled .hide-when-js-enabled {
 
 @media only screen and (min-width: 1280px) {
   #content {
-    width: 70vw;
+    width: 63vw;
```

Comment:
did you mean to revert 70 back to 63 here?

Reply:
Yes. I tried 70vw but it left almost no space for the left column with the synopsis and table of contents, and the lines became too long to read. I am now on a 1280px screen, therefore the `#content` has 806px, which seems pretty fine to me. I guess most people will view these pages using a 1920px screen in full or split mode. When split, the viewport will have less than 1280px and it will make more use of the screen, meaning that in most cases the use of space is pretty good.

Reply:
ah, didn't realize the changes made to the contents/synopsis to move them to the right. Yes, this is nice and fine.